src/components/ChatApp/Settings/Settings.css

```diff
@@ -70,6 +81,12 @@
   background-color : rgb(242, 242, 242) ;
 }
 
+.right-cheveron{
```

Comment:
Is there a reason why this is `cheveron` instead of `chevron` ? Please correct the typo and the related class usage